src/game/behaviors/door.inc.c

```diff
@@ -34,7 +34,7 @@ void set_door_camera_event(void) {
 }
 
 void play_door_open_noise(void) {
-    s32 isMetalDoor = cur_obj_has_model(MODEL_HMC_METAL_DOOR);
+    s32 isMetalDoor = (cur_obj_has_model(MODEL_HMC_METAL_DOOR) || cur_obj_has_model(MODEL_CASTLE_METAL_DOOR));
```

Comment:
useless parenthesis (must be destroyed)

Reply:
I actually prefer the parentheses in this situation since it's already a truthiness check sorta like an if statement. It's more unintuitive at a glance (although in general it would look nicer to just actually use an if statement despite not being 'optimal').